packages/venia-concept/src/components/Header/searchTrigger.js

```diff
@@ -21,7 +21,11 @@ class SearchTrigger extends Component {
 
         return (
             <Fragment>
-                <button className={searchClass} onClick={toggleSearch}>
+                <button
+                    className={searchClass}
+                    name="button"
```

Comment:
```suggestion
                    aria-label={'Search Button'}
```

Reply:
Thank you! I changed.

Reply:
I could be wrong about this but as I understand it:
`arial-label={'Search Button'}`

Should probably be:
`aria-label={'Search'}`

When an accessibility aid encounters `aria-label` it announces the label AND the element type for clarity. Including a description of the element in the label is an anti-pattern because it clutters the announcement with duplicate info.
> search button, button

The rewrite would be announced as intended:
> search, button

Aside: I'd love to be proven wrong if I've misunderstood this, but this article makes a clear case for not qualifying the element as part of the label:
https://knowbility.org/blog/2019/ARIA-labels/

Reply:
You're absolutely right, Brendan. I'm amending the commit to leave @khoa-le as author but slightly modify the wording.